src/features/view-nft/Actions/Actions.constants.ts

```diff
@@ -0,0 +1,12 @@
+export enum DataTestId {
+	CONTAINER = 'actions-container',
+	BUY_NOW = 'actions-buy-now',
+	SET_BUY_NOW = 'actions-set-buy-now',
+	BID = 'actions-bid',
+	USER_BID = 'actions-user-bid',
+}
+
+export enum Messages {
+	NO_BIDS_PLACED = 'No bids placed',
+	NO_BUY_NOW = 'No buy now set',
+}
```

Comment:
If these messages are used in one place I think it would be best to inline them. Bit easier to read in the component.

Reply:
Okay cool so in this case we can hard code? Shall take this on board moving forward :)